### PR TITLE
[IMP] theme_aviato: revamp the theme

### PR DESCRIPTION
**Pull request by xlu-odoo**
task-4178017
Part of task-4177975

### CODE REVIEW

File: theme_aviato/views/snippets/s_showcase.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt48 pb104" remove="pt48 pb48" separator=" "/>
+    </xpath>
+    <!-- Row -->
+    <xpath expr="//div[hasclass('container')]//div" position="attributes">
+        <attribute name="class" add="align-items-center" separator=" "/>
+    </xpath>
+
+    align-items-center
```
Comment (bvr-odoo) [inline theme_aviato/views/snippets/s_showcase.xml:14]: You forgot to remove this line

**Review by bvr-odoo — CHANGES_REQUESTED** [review]

File: theme_aviato/__manifest__.py [hunk 2]
```diff
@@ -56,14 +61,18 @@
         'static/description/aviato_screenshot.jpg',
     ],
     'images_preview_theme': {
-        'website.s_cover_default_image': '/theme_aviato/static/src/img/content/s_cover.jpg',
+        'website.s_image_text_default_image': '/theme_aviato/static/src/img/content/s_banner_2.jpg',
+        'website.s_product_list_default_image_1': '/theme_aviato/static/src/img/content/s_banner_3.jpg',
+        'website.s_picture_default_image': '/theme_aviato/static/src/img/content/s_popup.jpg',
+        'website.s_media_list_default_image_1': '/theme_aviato/static/src/img/content/s_wall_01.jpg',
```
Comment (bvr-odoo) [inline theme_aviato/__manifest__.py:67]: image
There are 3 times the same image on `s_media_list`.
It was already like this before this PR, but could you fix it please

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Review by bvr-odoo — APPROVED** [review]

**Comment by bvr-odoo** [comment]
@robodoo r+

**Comment by bvr-odoo** [comment]
@robodoo r-

**Comment by bvr-odoo** [comment]
@xlu-odoo needs to be rebased on 18.0 :wink:

**Comment by xlu-odoo** [comment]
@bvr-odoo sorry for all notifications you must have received this morning.. Runbot is green 👍

**Comment by qsm-odoo** [comment]
Being moved to https://github.com/odoo/design-themes/pull/969 (hopefully, with correct Signed-off-by)
Don't push here or there anymore, if you need to apply fixes, create a new PR to merge once the main one is merged.

**Pull request closed without merging**